cli/command/load_balancer_test.go

```diff
@@ -6,41 +6,323 @@ import (
 	"github.com/quintilesims/layer0/common/models"
 )
 
-func TestCreateLoadBalancer(t *testing.T) {
+func TestAddPortToLoadBalancer(t *testing.T) {
 	base, ctrl := newTestCommand(t)
 	defer ctrl.Finish()
 
-	ports := []models.Port{}
+	base.Resolver.EXPECT().
+		Resolve("load_balancer", "lname").
+		Return([]string{"lid"}, nil)
+
+	ports := []models.Port{
+		models.Port{
+			ContainerPort: 80,
+			HostPort:      80,
+			Protocol:      "tcp",
+		},
+	}
 	healthCheck := models.HealthCheck{}
 
+	base.Client.EXPECT().
+		ReadLoadBalancer("lid").
+		Return(&models.LoadBalancer{
+			HealthCheck:      healthCheck,
+			LoadBalancerID:   "lid",
+			LoadBalancerName: "lname",
+			Ports:            ports,
+		}, nil)
+
+	ports = append(ports, models.Port{
+		ContainerPort: 81,
+		HostPort:      81,
+		Protocol:      "tcp",
+	})
+
+	req := models.UpdateLoadBalancerRequest{
+		LoadBalancerID: "lid",
+		Ports:          &ports,
+		HealthCheck:    &healthCheck,
+	}
+
+	base.Client.EXPECT().
+		UpdateLoadBalancer(req).
+		Return("jid", nil)
+
+	base.Client.EXPECT().
+		ReadJob("jid").
+		Return(&models.Job{
```

Comment:
[nit] should probably put the job in a variable - this multi-line `Return()` statement is pretty hard to read